content/influxdb/v2.4/query-data/flux/exists.md

```diff
@@ -3,8 +3,8 @@ title: Check if a value exists
 seotitle: Use Flux to check if a value exists
 list_title: Exists
 description: >
-  Use the Flux `exists` operator to check if a record contains a key or if that
-  key's value is `null`.
+  Use the Flux `exists` operator to check if a row record contains a column or if
```

Comment:
Did you mean to remove "record" and just write "... if a row contains ..." here? Not sure if "row record" is a term we're using in the docs.

Reply:
It's a term we've started to use sparingly anytime row data is represented as a record (`r`) like in `filter`, `map`, and `reduce`.

Reply:
But it's something we can back off on as well.

Reply:
I don't have a strong opinion on it. I just wasn't sure if it was a typo. We've called the type of `r` a few different things in that situation, including "row", "record" and even "object" (at least in our code). I guess that readers should just understand that they're all the same thing.